app/javascript/widget/composables/useDarkMode.js

```diff
@@ -0,0 +1,32 @@
+import { computed } from 'vue';
+import { useStoreGetters } from 'dashboard/composables/store';
+
+/**
+ * Composable for handling dark mode.
+ * @returns {Object} An object containing computed properties and methods for dark mode.
+ */
+
+export function useDarkMode() {
+  const getters = useStoreGetters();
+  const darkMode = computed(() => getters['appConfig/getDarkMode'].value);
```

Comment:
You can use `useMapGetter` for this

Reply:
Resolved in https://github.com/chatwoot/chatwoot/pull/9949/commits/7ec528bd3220a832a74f76e213840a91039307d0